.build.yml

```diff
@@ -1,13 +1,32 @@
-image: archlinux
+image: alpine/edge
```

Comment:
Do we really need edge? Or is latest enough? Hm, might want edge if we keep grabbing the man pages from the package...

Reply:
This is mostly so we get the latest sway when we release, rather than remain tied to Alpine's release cycle.